examples/forbid-anonymous-operations/config.yml

```diff
@@ -0,0 +1,8 @@
+server:
+  cors:
+    origins:
+      - "https://studio.apollographql.com/"
+plugins:
+  # this plugin doesn't have any configuration
+  # mention it here and you're set!
+  apollographql.forbid-anonymous-operations:
```

Comment:
I'm trying to advocate for a consistent naming scheme around plugins/layers. I'm using reverse DNS for group and then adding a name for the plugin. So this would be:
`com.apollographql.forbid-anonymous-operations`

I've noticed that not everyone is doing this, but we should try to encourage either this standard or some other standard. Maybe we need a short/simple ADR to document this?

Reply:
fixed in def7ea6a

Reply:
definitely worth opening an ADR !

Reply:
opened https://github.com/apollographql/router/issues/581